web3_proxy/src/frontend/users.rs

```diff
@@ -757,13 +760,13 @@ pub async fn user_revert_logs_get(
 
     let db_replica = app
         .db_replica()
-        .context("getting replica db for user's revert logs")?;
+        .web3_context("getting replica db for user's revert logs")?;
 
     let uks = rpc_key::Entity::find()
         .filter(rpc_key::Column::UserId.eq(user.id))
         .all(db_replica.conn())
         .await
-        .context("failed loading user's key")?;
+        .web3_context("failed loading user's key")?;
```

Comment:
should this be a 5xx? this should never happen technically (though i guess it should be a 5xx, idk what else it would be)